doc/classes/EditorExportPlugin.xml

```diff
@@ -78,6 +78,14 @@
 			<description>
 			</description>
 		</method>
+		<method name="add_ios_embedded_framework">
+			<return type="void">
+			</return>
+			<argument index="0" name="path" type="String">
+			</argument>
+			<description>
```

Comment:
Could you add documentation for this method?

Reply:
Added it for both methods :)